### PR TITLE
[main] update fontconfig to v2.13.95

### PR DESCRIPTION
**Pull request by Camelron**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- update fontconfig to v2.13.95

###### Does this affect the toolchain?  

NO

###### Test Methodology

- Mock build

### CODE REVIEW

File: SPECS/fontconfig/fontconfig.spec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Summary:        library for configuring and customizing font access.
 Name:           fontconfig
-Version:        2.13.91
+Version:        2.13.95
 Release:        1%{?dist}
 License:        BSD/GPL
```
Comment (Camelron) [inline SPECS/fontconfig/fontconfig.spec:5]: Having trouble verifying the license for this one,

can someone make sense of the COPYING.md here? 
  https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/main/COPYING

Comment (Camelron) [reply SPECS/fontconfig/fontconfig.spec:5]: @christopherco any thoughts here?

Comment (christopherco) [reply SPECS/fontconfig/fontconfig.spec:5]: Hmm, looks to be a few licenses together.
- MIT overall
- fc-case/CaseFolding.txt = Unicode
- src/fcatomic.h = MIT
- src/fcfoundry.h = MIT
- src/fcmd5.h = Public Domain
- src/fcmutex.h = MIT
- src/ftglue.[ch] = Public Domain

**Review by christopherco — APPROVED** [review]

**Comment by christopherco** [comment]
Please do address linter findings

File: SPECS/fontconfig/fontconfig.spec [hunk 2]
```diff
@@ -68,20 +68,31 @@ make -k check
 %{_mandir}/man3/*
 
 %changelog
+* Wed Feb 02 2022 Cameron Baird <cameronbaird@microsoft.com> - 2.13.95-1
```
Comment (thcrain-msft) [inline SPECS/fontconfig/fontconfig.spec:71]: It won't let me comment on the offending line, but could you remove the `Provides: pkgconfig(fontconfig)` statement in the main package? It's both in the wrong place and not necessary.

File: SPECS/fontconfig/fontconfig.spec [hunk 3]
```diff
@@ -1,8 +1,8 @@
 Summary:        library for configuring and customizing font access.
 Name:           fontconfig
-Version:        2.13.91
+Version:        2.13.95
 Release:        1%{?dist}
-License:        BSD/GPL
+License:        MIT/Unicode/Public Domain
```
Comment (thcrain-msft) [inline SPECS/fontconfig/fontconfig.spec:5]: Please change the license line to separate licenses with `and` and `or`, so the overall licensing is clearer.

**Pull request merged**